src/connection/xmpp/Connector.ts

```diff
@@ -28,9 +30,9 @@ export default class Connector {
          let type = this.connectionParameters.get('type');
 
          if (type === TYPE.WEBSOCKET) {
-            throw new Error('Cannt attach to websocket connection.');
+            console.error('Can not attach to websocket connection.');
```

Comment:
Why did you remove the throw?

Reply:
when using websockets the exception what is thrown will totally cancel the login workflow... 
but websocket init will follow after that check...